test/Interop/Cxx/static/constexpr-static-member-var-errors.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=ConstexprStaticMemberVarErrors -I %S/Inputs -source-filename=x -enable-cxx-interop 2>&1 | %FileCheck %s
+
+// This test is simply testing that we properly report the error and don't crash
+// when importing an invalid decl.
```

Comment:
Nit: Please avoid using terms like "simply"; I don't think it's adding much... You could instead write:

```
// Check that we [...]
```

Reply:
Noted. Thanks :)